odoo/addons/base/models/res_partner.py

```diff
@@ -458,6 +459,10 @@ def _compute_company_registry(self):
         for company in self:
             company.company_registry = company.company_registry
 
+    def _compute_company_registry_placeholder(self):
+        for partner in self:
+            partner.company_registry_placeholder = ""
```

Comment:
Nitpicking, you can assign to all records at once.
```suggestion
        self.company_registry_placeholder = ""
```

Reply:
Do we really need to assign an empty value anyway? What about assigning `False` instead?

Reply:
Can be false, but you need to assign something in a computed field.